index.html

```diff
@@ -315,6 +315,52 @@ <h3>
             </div>
         </div>
     </section>
+    <!-- form section -->
+    <section class="form-section">
+        <article class="form-article">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </article>
+        <form novalidate action="https://formspree.io/f/xqkjrlry" method="post">
+
+            <input type="text" id="full-name" class="form-controll" maxlength="30" name="full_name"
+                placeholder="First name" required>
+            <input type="email" id="email" class="form-controll" name="user-email" placeholder="Email address"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
+            <textarea name="message" maxlength="500" placeholder="Enter message here" required></textarea>
+            <button class="form_button" type="submit">Get in touch</button>
+        </form>
+    </section>
```

Comment:
Good job! but It's highly recommended to follow the exact Figma template, so please make sure to add some space to the right side of your form elements as you have it on the left side.

![image](https://user-images.githubusercontent.com/34779928/206285738-bc1e35c1-4271-478f-b3c2-48c46322a181.png)